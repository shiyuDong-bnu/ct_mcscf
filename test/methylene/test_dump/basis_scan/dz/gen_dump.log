2025-05-11:10:13:45,884 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'DF_BASIS_MP2': 'CC-PVDZ-F12-OPTRI', 'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'FREEZE_CORE': '1', 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'SCREENING': 'CSAM', 'forte__scf_type': 'PK', 'forte__e_convergence': 1e-10, 'forte__d_convergence': 1e-08, 'forte__reference': 'RHF', 'forte__maxiter': 300, 'forte__basis': 'CC-PVDZ-F12', 'forte__df_basis_mp2': 'CC-PVDZ-F12-OPTRI', 'function_kwargs': {}}
2025-05-11:10:13:45,887 INFO     [psi4.driver.driver:523] Compute energy(): method=scf, basis=cc-pvdz-f12, molecule=default, nre=6.014699644935847
2025-05-11:10:13:46,367 INFO     [psi4.driver.driver:526] Return energy(): -38.89230504250601
2025-05-11:10:13:46,550 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'DF_BASIS_MP2': 'CC-PVDZ-F12-OPTRI', 'D_CONVERGENCE': 1e-08, 'E_CONVERGENCE': 1e-10, 'FREEZE_CORE': '1', 'SCF__INTS_TOLERANCE': 1e-12, 'MAXITER': 300, 'REFERENCE': 'RHF', 'SCF_TYPE': 'PK', 'SCREENING': 'CSAM', 'forte__scf_type': 'PK', 'forte__e_convergence': 1e-10, 'forte__d_convergence': 1e-08, 'forte__reference': 'RHF', 'forte__maxiter': 300, 'forte__basis': 'CC-PVDZ-F12', 'forte__df_basis_mp2': 'CC-PVDZ-F12-OPTRI', 'function_kwargs': {}}
2025-05-11:10:13:46,552 INFO     [psi4.driver.driver:523] Compute energy(): method=forte, basis=cc-pvdz-f12, molecule=default, nre=6.014699644935847
2025-05-11:10:13:49,711 INFO     [psi4.driver.driver:526] Return energy(): -38.95361423038886
