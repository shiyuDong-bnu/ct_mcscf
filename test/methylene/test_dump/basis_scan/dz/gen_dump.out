
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.9.1 release

                         Git: Rev {HEAD} f53cdd7 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, M. H. Lechner, A. Jiang, S. Behnle, A. G. Heide,
    M. F. Herbst, and D. L. Poole

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Sunday, 11 May 2025 10:13AM

    Process ID: 3993168
    Host:       hydrogen
    PSIDATADIR: /data/home/sydong/opt/psi4/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    Molecule: geometry: Molecule is not complete, please use 'update_geometry'
                    once all variables are set.
Molecule: Setting geometry variable r_ch to 2.102300
Molecule: Setting geometry variable ahch to 101.710000

Scratch directory: /data/home/sydong/scratch/psi4/
   => Libint2 <=

    Primary   basis highest AM E, G, H:  6, 6, 3
    Auxiliary basis highest AM E, G, H:  7, 7, 4
    Onebody   basis highest AM E, G, H:  -, -, -
    Solid Harmonics ordering:            Gaussian

*** tstart() called on hydrogen
*** at Sun May 11 10:13:45 2025

   => Loading Basis Set <=

    Name: CC-PVDZ-F12
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry C          line   170 file /data/home/sydong/opt/psi4/share/psi4/basis/cc-pvdz-f12.gbs 
    atoms 2-3 entry H          line    12 file /data/home/sydong/opt/psi4/share/psi4/basis/cc-pvdz-f12.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C2v

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         C            0.000000000000     0.000000000000    -0.190863170104    12.000000000000
         H            0.000000000000    -1.630440522826     1.136287534046     1.007825032230
         H            0.000000000000     1.630440522826     1.136287534046     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A =     19.80481  B =     11.23490  C =      7.16840 [cm^-1]
  Rotational constants: A = 593733.17106  B = 336813.79877  C = 214903.20347 [MHz]
  Nuclear repulsion =    6.014699644935847

  Charge       = 0
  Multiplicity = 1
  Electrons    = 8
  Nalpha       = 4
  Nbeta        = 4

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-10
  Density threshold  = 1.00e-08
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: CC-PVDZ-F12
    Blend: CC-PVDZ-F12
    Number of shells: 22
    Number of basis functions: 48
    Number of Cartesian functions: 50
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   3
      Number of AO shells:              22
      Number of primitives:             41
      Number of atomic orbitals:        50
      Number of basis functions:        48

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 1384152 doubles for integral storage.
  We computed 32131 shell quartets total.
  Whereas there are 32131 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 3.1200075401E-03.
  Reciprocal condition number of the overlap matrix is 5.5294913187E-04.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A         48      48 
   -------------------------
    Total      48      48
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:   -38.15612299453069   -3.81561e+01   0.00000e+00 
   @RHF iter   1:   -38.87945122910550   -7.23328e-01   2.70278e-03 ADIIS/DIIS
   @RHF iter   2:   -38.89146258122322   -1.20114e-02   6.77072e-04 ADIIS/DIIS
   @RHF iter   3:   -38.89220018713830   -7.37606e-04   1.89164e-04 ADIIS/DIIS
   @RHF iter   4:   -38.89230033057817   -1.00143e-04   4.06457e-05 DIIS
   @RHF iter   5:   -38.89230480796007   -4.47738e-06   7.98811e-06 DIIS
   @RHF iter   6:   -38.89230504020703   -2.32247e-07   7.60843e-07 DIIS
   @RHF iter   7:   -38.89230504240127   -2.19423e-09   1.61256e-07 DIIS
   @RHF iter   8:   -38.89230504250366   -1.02389e-10   2.23643e-08 DIIS
   @RHF iter   9:   -38.89230504250601   -2.35190e-12   2.80159e-09 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A    -11.294851     2A     -0.892023     3A     -0.563553  
       4A     -0.396785  

    Virtual:                                                              

       5A      0.025211     6A      0.071587     7A      0.091461  
       8A      0.112599     9A      0.131442    10A      0.236712  
      11A      0.249083    12A      0.469605    13A      0.511518  
      14A      0.566557    15A      0.581956    16A      0.592445  
      17A      0.622414    18A      0.640533    19A      0.678080  
      20A      0.692547    21A      0.905078    22A      1.047336  
      23A      1.087516    24A      1.088562    25A      1.117266  
      26A      1.246480    27A      1.353909    28A      1.463228  
      29A      2.223411    30A      2.381881    31A      2.528290  
      32A      2.634970    33A      2.719495    34A      2.798070  
      35A      2.817298    36A      2.875636    37A      3.085097  
      38A      3.185367    39A      3.197013    40A      3.390212  
      41A      3.392160    42A      3.560846    43A      3.945784  
      44A      4.331107    45A      6.645570    46A      9.183110  
      47A      9.520196    48A      9.706291  

    Final Occupation by Irrep:
              A 
    DOCC [     4 ]
    NA   [     4 ]
    NB   [     4 ]

  @RHF Final Energy:   -38.89230504250601

   => Energetics <=

    Nuclear Repulsion Energy =              6.0146996449358472
    One-Electron Energy =                 -63.5195853602797982
    Two-Electron Energy =                  18.6125806728379466
    Total Energy =                        -38.8923050425060097

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix


 Multipole Moments:

 ------------------------------------------------------------------------------------
     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)
 ------------------------------------------------------------------------------------

 L = 1.  Multiply by 2.5417464519 to convert [e a0] to [Debye]
 Dipole X            :          0.0000000            0.0000000            0.0000000
 Dipole Y            :         -0.0000000            0.0000000           -0.0000000
 Dipole Z            :         -0.3622895            1.1273960            0.7651065
 Magnitude           :                                                    0.7651065

 ------------------------------------------------------------------------------------

*** tstop() called on hydrogen at Sun May 11 10:13:46 2025
Module time:
	user time   =       0.38 seconds =       0.01 minutes
	system time =       0.08 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       0.38 seconds =       0.01 minutes
	system time =       0.08 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

Scratch directory: /data/home/sydong/scratch/psi4/

  Forte
  ----------------------------------------------------------------------------
  A suite of quantum chemistry methods for strongly correlated electrons

    git branch: ct_forte - git commit: 41562031

  Developed by:
    Francesco A. Evangelista, Chenyang Li, Kevin P. Hannon,
    Jeffrey B. Schriber, Tianyuan Zhang, Chenxi Cai,
    Nan He, Nicholas Stair, Shuhe Wang, Renke Huang
  ----------------------------------------------------------------------------

  Forte will use options passed as a dictionary. Option read from psi4 will be ignored


  Preparing forte objects from a Psi4 Wavefunction object Using basis set CC-PVDZ-F12 from Psi4 global options.   => Loading Basis Set <=

    Name: CC-PVDZ-F12
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry C          line   170 file /data/home/sydong/opt/psi4/share/psi4/basis/cc-pvdz-f12.gbs 
    atoms 2-3 entry H          line    12 file /data/home/sydong/opt/psi4/share/psi4/basis/cc-pvdz-f12.gbs 



  ==> MO Space Information <==

  -------------------------------
                        A   Sum
  -------------------------------
    FROZEN_DOCC         1     1
    RESTRICTED_DOCC     0     0
    GAS1                6     6
    GAS2                0     0
    GAS3                0     0
    GAS4                0     0
    GAS5                0     0
    GAS6                0     0
    RESTRICTED_UOCC    41    41
    FROZEN_UOCC         0     0
    Total              48    48
  -------------------------------   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: MINAO_BASIS
    atoms 1   entry C          line    61 file /data/home/sydong/opt/psi4/share/psi4/basis/sto-3g.gbs 
    atoms 2-3 entry H          line    19 file /data/home/sydong/opt/psi4/share/psi4/basis/sto-3g.gbs 


  State Singlet (Ms = 0) A GAS min: 0 0 0 0 0 0 ; GAS max: 12 0 0 0 0 0 ; weights:
      1.000000000000
  Forte will use psi4 integrals

  ==> Primary Basis Set Summary <==

  Basis Set: CC-PVDZ-F12
    Blend: CC-PVDZ-F12
    Number of shells: 22
    Number of basis functions: 48
    Number of Cartesian functions: 50
    Spherical Harmonics?: true
    Max angular momentum: 2


  JK created using conventional PK integrals
  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   3
      Number of AO shells:              22
      Number of primitives:             41
      Number of atomic orbitals:        50
      Number of basis functions:        48

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 1384152 doubles for integral storage.
  We computed 32131 shell quartets total.
  Whereas there are 32131 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              400
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1



  ==> Integral Transformation <==

  Number of molecular orbitals:                         48
  Number of correlated molecular orbitals:              47
  Number of frozen occupied orbitals:                    1
  Number of frozen unoccupied orbitals:                  0
  Two-electron integral type:                 Conventional


  Skip integral allocation and transformation for AO-driven CASSCF.

          -----------------------------------------------------------
                  Multi-Configurational Self Consistent Field
                Two-Step Approximate Second-Order AO Algorithm
            written by Chenyang Li, Kevin P. Hannon, and Shuhe Wang
          -----------------------------------------------------------


  ==> MCSCF Calculation Information <==

    --------------------------------------------------------
    Print level                                      Default
    Integral type                               CONVENTIONAL
    CI solver type                                       FCI
    Final orbital type                             CANONICAL
    Derivative type                                     NONE
    Optimize orbitals                                   TRUE
    Include internal rotations                         FALSE
    Debug printing                                     FALSE
    Energy convergence                             1.000e-08
    Gradient convergence                           1.000e-07
    Max value for rotation                         2.000e-01
    Max number of macro iter.                            100
    Max number of micro iter. for orbitals                 6
    Max number of micro iter. for CI                      12
    DIIS start                                            15
    Min DIIS vectors                                       3
    Max DIIS vectors                                       8
    Frequency of DIIS extrapolation                        1
    --------------------------------------------------------

  ==> Independent Orbital Rotations <==

    ORBITAL SPACES                         A
    ----------------------------------------
             ACTIVE / RESTRICTED_DOCC      0
    RESTRICTED_UOCC /          ACTIVE    246
    RESTRICTED_UOCC / RESTRICTED_DOCC      0
    ----------------------------------------

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              3
    number of beta electrons                               3
    number of alpha strings                               20
    number of beta strings                                20
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                               400
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------
  Initial guess space is incomplete.
  Adding 8 determinant(s).

  ==> Initial Guess <==

  Initial guess determinants:         58

  Classification of the initial guess solutions

  Number   2S+1   Selected
  ------------------------
    27       1       *
    27       3        
     4       5        
  ------------------------

    Spin    Root           Energy        <S^2>    Status
  -------------------------------------------------------
   singlet    0      -38.893958085747  -0.000000  added
  -------------------------------------------------------

  ==> Davidson-Liu Solver <==

    --------------------------------------------------------
    Print level                                      Default
    Energy convergence threshold                   1.000e-08
    Residual convergence threshold                 1.000e-06
    Schmidt orthogonality threshold                1.000e-12
    Schmidt discard threshold                      1.000e-07
    Size of the space                                    400
    Number of roots                                        1
    Maximum number of iterations                         100
    Collapse subspace size                                 2
    Maximum subspace size                                 10
    --------------------------------------------------------

  Davidson-Liu solver: adding 1 guess vectors
  Iteration     Average Energy            max(∆E)            max(Residual)  Vectors
  ---------------------------------------------------------------------------------
       0      -38.893958085747       38.893958085747        0.039948300711      1
       1      -38.895076406410        0.001118320663        0.005353447722      2
       2      -38.895113375234        0.000036968824        0.001424876757      3
       3      -38.895115423576        0.000002048342        0.000217970579      4
       4      -38.895115461753        0.000000038177        0.000030706376      5
       5      -38.895115462329        0.000000000576        0.000003369508      6
       6      -38.895115462334        0.000000000005        0.000000294669      7
  ---------------------------------------------------------------------------------

  ==> Root No. 0 <==

    222000      0.99829416
    220200     -0.04380687
    022200     -0.02721473
    b2a200      0.01605320
    a2b200      0.01605320
    202200     -0.01015318

    Total Energy:     -38.895115462334, <S^2>: 0.000000
    Time for FCI:       0.017659607000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0      -38.895115462334   0.000000
    --------------------------------------------------------
  Computing RDMs for dipole moments ... Done

  ==> Summary of Dipole Moments [e a0] (Nuclear + Electronic) <==

       State           DM_X           DM_Y           DM_Z           |DM|
    --------------------------------------------------------------------
          0A     0.00000000    -0.00000000     0.76017474     0.76017474
    --------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     1.12739605     1.12739605
    --------------------------------------------------------------------

  ==> Natural Orbitals <==

        2A      1.999687      3A      1.999250      4A      1.994248  
        5A      0.006563      6A      0.000151      7A      0.000100  

 sydong:  unsymmetrize 2RDM


  ==> MCSCF Iterations <==

                      Energy CI                    Energy Orbital
           ------------------------------  ------------------------------
    Iter.        Total Energy       Delta        Total Energy       Delta  Orb. Grad.  Micro
    ----------------------------------------------------------------------------------------
 sydong:  unsymmetrize 2RDM

  L-BFGS Warning: Skip this vector due to negative rho
  L-BFGS Warning: Skip this vector due to negative rho
  L-BFGS Warning: Skip this vector due to negative rho
       1     -38.895115462334 -3.8895e+01    -38.908420706548 -3.8908e+01  9.2368e-04    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       2     -38.941903313797 -4.6788e-02    -38.949384540253 -4.0964e-02  1.1247e-03    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       3     -38.952521929465 -1.0619e-02    -38.953240901501 -3.8564e-03  2.2192e-05    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       4     -38.953494004064 -9.7207e-04    -38.953571717804 -3.3082e-04  7.1854e-06    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       5     -38.953599183366 -1.0518e-04    -38.953608756982 -3.7039e-05  3.2893e-06    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       6     -38.953612229379 -1.3046e-05    -38.953613491070 -4.7341e-06  1.2670e-06    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       7     -38.953613955666 -1.7263e-06    -38.953614127712 -6.3664e-07  4.7678e-07    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       8     -38.953614191817 -2.3615e-07    -38.953614215870 -8.8158e-08  1.7849e-07    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

       9     -38.953614224935 -3.3119e-08    -38.953614228378 -1.2508e-08  6.7127e-08    6/N

  Davidson-Liu solver: restarting from previous calculation.
 sydong:  unsymmetrize 2RDM

      10     -38.953614229690 -4.7547e-09    -38.953614230194 -1.8159e-09  2.5586e-08    6/N
    ----------------------------------------------------------------------------------------

  A miracle has come to pass: MCSCF iterations have converged!

  Performing final CI Calculation using converged orbitals

  ==> String Lists <==

    --------------------------------------------------------
    number of alpha electrons                              3
    number of beta electrons                               3
    number of alpha strings                               20
    number of beta strings                                20
    --------------------------------------------------------

  ==> FCI Solver <==

    --------------------------------------------------------
    Spin adapt                                         FALSE
    Number of determinants                               400
    Symmetry                                               0
    Multiplicity                                           1
    Number of roots                                        1
    Target root                                            0
    --------------------------------------------------------

  ==> Davidson-Liu Solver <==

    --------------------------------------------------------
    Print level                                      Default
    Energy convergence threshold                   1.000e-08
    Residual convergence threshold                 1.000e-06
    Schmidt orthogonality threshold                1.000e-12
    Schmidt discard threshold                      1.000e-07
    Size of the space                                    400
    Number of roots                                        1
    Maximum number of iterations                         100
    Collapse subspace size                                 2
    Maximum subspace size                                 10
    --------------------------------------------------------

  Davidson-Liu solver: restarting from previous calculation.
  Iteration     Average Energy            max(∆E)            max(Residual)  Vectors
  ---------------------------------------------------------------------------------
       0      -38.953614230194        0.000000000504        0.000016850767      1
       1      -38.953614230374        0.000000000180        0.000004926950      2
       2      -38.953614230387        0.000000000013        0.000001613463      3
       3      -38.953614230389        0.000000000001        0.000000309729      4
  ---------------------------------------------------------------------------------

  ==> Root No. 0 <==

    222000      0.96714394
    220200     -0.17969285
    202002     -0.07134394
    b2a200      0.06502152
    a2b200      0.06502152
    022200     -0.04388826
    022020     -0.04303888
    ba20ab      0.04023655
    ab20ba      0.04023655
    202020     -0.03629494
    2ba0ba      0.03331481
    2ab0ab      0.03331481
    b2a020     -0.03069433
    a2b020     -0.03069433
    ab20ab      0.02920324
    ba20ba      0.02920324
    220002     -0.02747450
    220020     -0.02437246
    2ab0ba      0.02292855
    2ba0ab      0.02292855
    022002     -0.02217974
    200202      0.01356944
    020220      0.01317498
    bb20aa     -0.01103330
    aa20bb     -0.01103330
    2bb0aa      0.01038626
    2aa0bb      0.01038626

    Total Energy:     -38.953614230389, <S^2>: -0.000000
    Time for FCI:       0.010715414000

  ==> Energy Summary <==

    Multi.(2ms)  Irrep.  No.               Energy      <S^2>
    --------------------------------------------------------
       1  (  0)     A     0      -38.953614230389  -0.000000
    --------------------------------------------------------
  Computing RDMs for dipole moments ... Done

  ==> Summary of Dipole Moments [e a0] (Nuclear + Electronic) <==

       State           DM_X           DM_Y           DM_Z           |DM|
    --------------------------------------------------------------------
          0A     0.00000000     0.00000000     0.65010352     0.65010352
    --------------------------------------------------------------------
     Nuclear     0.00000000     0.00000000     1.12739605     1.12739605
    --------------------------------------------------------------------

  ==> Natural Orbitals <==

        2A      1.981419      3A      1.977102      4A      1.910382  
        5A      0.087370      6A      0.022626      7A      0.021101  

  Canonicalizing final MCSCF orbitals

  ==> Semicanonicalize Orbitals <==

    MIX INACTIVE ORBITALS         FALSE
    MIX GAS ACTIVE ORBITALS       FALSE
    FROZEN_DOCC                   CANONICAL
    FROZEN_UOCC                   CANONICAL
    GAS1                          CANONICAL
    RESTRICTED_DOCC               CANONICAL
    RESTRICTED_UOCC               CANONICAL

    Off-Diag. Elements       Max           2-Norm
    ------------------------------------------------
    FROZEN_DOCC          0.0000000000   0.0000000000
    GAS1                 0.0081571909   0.0133804888
    RESTRICTED_UOCC      0.3851606304   1.3762834483
    ------------------------------------------------

    Canonicalization test failed

  Warning: Failed to fix orbital phase and order.
  Timing for orbital canonicalization:                        0.000 s.

  !!! WARNING: Final Active Orbitals Maybe Different from the Original !!!

    Based on projections of maximum overlap method:
           5A -> 8A        RESTRICTED_UOCC
           6A -> 9A        RESTRICTED_UOCC
  Orbital rotations on spin-free RDMs ...
    Transformed 1 RDMs.
    Transformed 2 RDMs.

  Time to prepare integrals:        0.190 seconds
  Time to run job          :        2.965 seconds
  Total                    :        3.155 seconds
    Orthogonalizing basis for space RIBS.
    0 linear dependencies will be 'removed'.
    Projecting out 'ALL' from 'RIBS' to obtain space 'CABS'
    0 linear dependencies will be 'removed'.
        Orbital space before projecting out:    (n = 1): 158  
        Orbital space after projecting out:     (n = 1): 110  

*********************************************************************************************************************
When computing potential integrals with different bra and ket basis, the atom definition is taken from the bra basis.
*********************************************************************************************************************
*********************************************************************************************************************
When computing potential integrals with different bra and ket basis, the atom definition is taken from the bra basis.
*********************************************************************************************************************
